src/main/java/com/cognifide/aemrules/java/checks/ThreadSafeFieldCheck.java

```diff
@@ -56,24 +55,24 @@
 
     public static final String RULE_MESSAGE = "Usage of %s as a field is not thread safe.";
 
-    private static Set<String> vulnerableClasses = Sets.newHashSet(
+    private static final Set<String> vulnerableClasses = Set.of(
```

Comment:
If we're making this `final`, let's also update the naming convention.

Reply:
naming convention with uppercase is for when combined with `static`

Reply:
fixed